src/main/java/uk/gov/hmcts/reform/domain/common/Party.java

```diff
@@ -9,14 +9,19 @@
 @AllArgsConstructor
 public class Party {
     private final String partyID;
-    private final String idamID;
     private final String partyType;
     private final String title;
     private final String firstName;
     private final String lastName;
-    private final String organisationName;
     private final String dateOfBirth;
     private final Address address;
-    private final EmailAddress email;
     private final TelephoneNumber telephoneNumber;
+    private final EmailAddress email;
+    private final String gender;
+    private final String genderIdentification;
+    private final String placeOfBirth;
+    private final String relationshipToChild;
```

Comment:
relationshipToChild is part of respondent, not party. Should we create a base party class that includes party properties only and extend?

Reply:
No. This branch will never go onto master, so it is fine for my own version of Party to exist. The data structure we are matching includes everything within party.

We should have proper classes that extend party in fpl-ccd-configuration... but here I do not see the need